packages/poaps/src/queries/utils.ts

```diff
@@ -29,6 +29,13 @@ export function creatEqFilter(
   return value ? { [key]: { _eq: value } } : {};
 }
 
+export function creatAddressFilter(
+  key: string,
+  value?: string,
+): Record<string, any> {
+  return value ? { [key]: { _eq: value.toLocaleLowerCase() } } : {};
```

Comment:
Why use environment locale for lower case? Addresses are all ascii so it makes no difference but the check for locale chars will never succeed and probably just wasted resources.